apps/emqx/src/emqx_authentication.erl

```diff
@@ -673,7 +661,7 @@ do_authenticate(
                 _ ->
                     ok
             end,
-            {stop, Result}
+            {ok, Result}
```

Comment:
shouldn't we stop if result is error ?

Reply:
@zmstone Yes if we want to forbid to override `emqx_authentication` result. Changed the behaviour to stop.